Keas.Mvc/Controllers/Api/EquipmentController.cs

```diff
@@ -63,6 +63,38 @@ select new
             return Json(await equipment.ToListAsync());
         }
 
+        [HttpGet]
+        [ProducesResponseType(typeof(IEnumerable<Equipment>), StatusCodes.Status200OK)]
+        public async Task<IActionResult> SearchAttributes(string q)
+        {
+            var attributes = from ea in _context.EquipmentAttributes
+                    .Where(a => a.Equipment.Team.Slug == Team && a.Equipment.Active &&
+                                (EF.Functions.Like(a.Value, q.EfStartsWith())))
+                    .Include(a => a.Equipment)
+                    .AsNoTracking()
+                select new
+                {
+                    ea.Equipment.Id
+                };
+            var ids = await attributes.ToListAsync();

```

Comment:
You should be able to call `Distinct()` before `ToListAsync()` so that it is performed at the db instead of locally.